lib/client/mfa/cli.go

```diff
@@ -204,6 +199,8 @@ func (c *CLIPrompt) Run(ctx context.Context, chal *proto.MFAAuthenticateChalleng
 	case promptSSO:
 		resp, err := c.promptSSO(ctx, chal)
 		return resp, trace.Wrap(err)
+	case isPerSessionMFA && promptOTP:
```

Comment:
Could you also add tests for these two modified `switch` branches in `cli_test.go`? We'd probably need test cases for

* otp + webauthn + session MFA
* otp + session MFA
* webauthn + session MFA
* otp + webauthn + not session MFA